flag_test.go

```diff
@@ -400,10 +400,14 @@ func testParse(newFlagSet func() *FlagSet, t *testing.T) {
 		}
 		err := f.Parse(g.intersperse, g.args)
 		if g.error != "" {
+			expectedError := g.error
+			if strings.Contains(expectedError, "%v") {
```

Comment:
This seems ~ ok. I'm a little concerned because it doesn't ever allow an escaping mechanism (%%v doesn't work because that still contains %v). we *could* do something like format/https://golang.org/pkg/text/template/ but that all feels like overkill.
We can always do that as soon as we need >1 object to be formatted.

Reply:
Let's worry about this use case when we come across it? :)